src/current/_includes/releases/v24.1/v24.1.0-alpha.4.md

```diff
@@ -54,7 +54,7 @@ Release Date: March 25, 2024
 
 <h3 id="v24-1-0-alpha-4-command-line-changes">Command-line changes</h3>
 
-- The new `--go-gc-percent` flag of the [`cockroach start`](../v24.1/cockroach-start.html) command controls the garbage collection target percentage of the Go runtime, mirroring the existing `GOGC` environment variable. A garbage collection is triggered when the ratio of freshly allocated data to live data remaining after the previous collection reaches this percentage. If left unspecified and if a Go soft memory limit is configured (i.e., not explicitly disabled via `--max-go-memory` or `GOMEMLIMIT`), the garbage collection target percentage defaults to 300%. Setting the flag to a negative value disables  the target percentage garbage collection heuristic, and only the soft memory limit heuristic triggers garbage collection. [#119605][#119605]
+- The new `--go-gc-percent` flag of the [`cockroach start`](../v24.1/cockroach-start.html) command controls the garbage collection target percentage of the Go runtime, mirroring the existing `GOGC` environment variable. A garbage collection is triggered when the ratio of freshly allocated data to live data remaining after the previous collection reaches this percentage. If left unspecified and if a Go soft memory limit is configured (i.e., not explicitly disabled via `--max-go-memory` or `GOMEMLIMIT`), the garbage collection target percentage defaults to 300%. Setting the flag to a negative value disables the target percentage garbage collection heuristic, and only the soft memory limit heuristic triggers garbage collection. From the user's perspective, this change will look like an increase in **Memory usage** in the [Hardware dashboard]({% link v24.1/ui-hardware-dashboard.md %}#memory-usage) and an increase in **Go total memory usage** in the [Runtime dashboard]({% link v24.1/ui-runtime-dashboard.md %}#memory-usage). This should not cause OOMs, as the Go memory limit (controlled by the `--max-go-memory` flag or the `GOMEMLIMIT` environment variable) will still prevent Go memory from growing too large. [#119605][#119605]
```

Comment:
```suggestion
- The new `--go-gc-percent` flag of the [`cockroach start`](../v24.1/cockroach-start.html) command controls the garbage collection target percentage of the Go runtime, mirroring the existing `GOGC` environment variable. A garbage collection is triggered when the ratio of freshly allocated data to live data remaining after the previous collection reaches this percentage. If left unspecified and if a Go soft memory limit is configured (i.e., not explicitly disabled via `--max-go-memory` or `GOMEMLIMIT`), the garbage collection target percentage defaults to 300%. Setting the flag to a negative value disables the target percentage garbage collection heuristic, and only the soft memory limit heuristic triggers garbage collection. To monitor the impact of this change in the DB Console, look for an increase in **Memory usage** in the [Hardware dashboard]({% link v24.1/ui-hardware-dashboard.md %}#memory-usage) and an increase in **Go total memory usage** in the [Runtime dashboard]({% link v24.1/ui-runtime-dashboard.md %}#memory-usage). This does not increase the risk of an out-of-memory exception (OOM), because the Go memory limit (controlled by the `--max-go-memory` flag or the `GOMEMLIMIT` environment variable) prevents Go from consuming too much memory. [#119605][#119605]
```
Just a suggestion, which I don't feel super strongly about but might be more direct.